app/config/ApplicationConfig.scala

```diff
@@ -106,4 +107,9 @@ class ApplicationConfig @Inject()(
   def payeFallbackURL: String = config.getString("paye.language.fallbackUrl")
 
   def saFallbackURL: String = config.getString("sa.language.fallbackUrl")
+
+  val saYear: Int = config.getInt("sa.year")
+
+  def spendCategories(taxYear: Int): Option[List[String]] =
+    Option(configuration.underlying.getStringList(s"categoryOrder.$taxYear").asScala.toList)
```

Comment:
Use configuration.get[A], which I believe doesn't throw but its something you can quickly check!